### PR TITLE
Update Help Command and update UG for Help and Find

### PR DESCRIPTION
**Pull request by bchenghi**
- Update help link
- Update help and find commands in User Guide

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -60,24 +66,20 @@ Format: `list`
 
 ### Searching for a food item: `find`
 
-Searches for a food item inside the inventory tracker according to `KEYWORD`, `PRIORITY` or `EXPIRY_DATE`.
+Searches inside the inventory tracker for food items which the descriptions contain any of the given keywords.
```
Comment (jonahtanjz) [inline docs/UserGuide.md]: Perhaps can consider "Searches for food items in the inventory with descriptions matching any of the given keywords." instead?

**Review by jonahtanjz — COMMENTED** [review]
Just 1 suggestion, everything else looks good!

File: docs/UserGuide.md (hunk 1 above)
Comment (bchenghi) [reply docs/UserGuide.md]: I agree with you, thank you!

**Review by jonahtanjz — APPROVED** [review]
LGTM

**Pull request merged**